packages/manager-react-components/src/hooks/datagrid/useResourcesV6.tsx

```diff
@@ -100,7 +99,7 @@ export function useResourcesV6({
 
     if (sortData && sorting) {
       const type = columns?.find((element) => element.id === sorting.id)?.type;
-      const sortedDatas = sortData?.sort((a, b) =>
+      const sortedDatas = sortData?.sort((a: any, b: any) =>
         sortColumn(type, a?.[sorting.id], b?.[sorting.id], sorting?.desc),
```

Comment:
type is a reserved word for Typescript. I think it's better to not use it as props name

```suggestion
        sortColumn(sortType, a?.[sorting.id], b?.[sorting.id], sorting?.desc),
```

Reply:
I don't find it confusing here, personally
And vscode colors it correctly for me